.gitignore

```diff
@@ -21,3 +22,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
+yarn.lock
```

Comment:
Didn't we say we didn't want to gitignore yarn.lock on my PR (#11)?

Reply:
Lmao

Reply:
I stand corrected

Reply:
Bamboozled. So should we still gitignore or not? If we should, then I'll delete the `yarn.lock` from my PR. if we don't, then you should remove this

Reply:
Hmmm, I'd say delete it then from your branch